lib/logstash/filters/uuid.rb

```diff
@@ -45,13 +45,12 @@ def filter(event)
     return unless filter?(event)
 
     if overwrite
-      event[target] = SecureRandom.uuid
+      event[target] = SecureRandom.uuid.force_encoding("UTF-8")
```

Comment:
Don't you think, this might be solved with something like https://github.com/elastic/logstash/pull/3723 ?